src/components/RulesSelector.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
 import RuleLabel from './RuleLabel';
 
-const makeListItems = (filters, onRuleSelected) => {
-  return Object.keys(filters)
+const makeListItems = (filters, onRuleSelected) =>
```

Comment:
I have no problem with removing the `{}`, but I think it would be best to replace them with `()`. Otherwise it can be slightly jarring to figure out the start/end of the block, since it does not all fit on one line.